srcpkgs/v2raya/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'v2raya'
+pkgname=v2raya
+version=1.5.9.1698.1
+revision=1
+wrksrc="v2rayA-${version}"
+build_style="go"
+go_import_path="github.com/v2rayA/v2rayA"
+go_ldflags='-X github.com/v2rayA/v2rayA/conf.Version=unstable-'"$version"' -s -w'
+hostmakedepends="go nodejs yarn"
+depends="v2ray>=4.37.0_1"
+short_desc="Web GUI client of Project V which supports various protocals"
+maintainer="ketlrznt <tansuanyinliao8888@gmail.com>"
+license="AGPL-3.0"
+homepage="https://github.com/v2rayA/v2rayA"
+distfiles="https://github.com/v2rayA/v2rayA/archive/v${version}.tar.gz"
+checksum="247a357230c616bf48309c61d119686e4ad56939c05afef584c45051e9dc6220"
+
+export GO11MODULE=on
+
+do_build() {
+	cd gui
+	yarn --check-files
+	OUTPUT_DIR="${wrksrc}/service/server/router/web" yarn build
+	cd ../service
+	go build -ldflags="$go_ldflags" -o "${GOPATH}/bin/v2raya"
```

Comment:
```suggestion
pre_build() {
	cd gui
	yarn --check-files
	OUTPUT_DIR="${wrksrc}/service/server/router/web" yarn build
```
there's no reason to override the go build style, i think

Reply:
also it seems the yarn build output isn't installed?

Reply:
https://github.com/v2rayA/v2rayA/blob/v1.5.9.1698.1/service/server/router/index.go#L27-L28

yarn build output is embed in `v2raya` with `go:embed` feature.